diff --git a/node_modules/adbkit/index.js b/node_modules/adbkit/index.js
index a4c08ac..dca38d0 100644
--- a/node_modules/adbkit/index.js
+++ b/node_modules/adbkit/index.js
@@ -4,12 +4,7 @@
   Path = require('path');
 
   module.exports = (function() {
-    switch (Path.extname(__filename)) {
-      case '.coffee':
-        return require('./src/adb');
-      default:
-        return require('./lib/adb');
-    }
+    return require('./lib/adb');
   })();
 
 }).call(this);
